src/apply_manual_codes.py

```diff
@@ -187,4 +187,15 @@ def apply_manual_codes(cls, user, data, coda_input_dir):
                         Metadata.get_call_location()).to_dict()
                 }, Metadata(user, Metadata.get_call_location(), time.time()))
 
+            # If the location is not coded, set the zone from the operator
```

Comment:
Would it be easy to factor this into another function? I'm interested in taking things which do logical imputation and moving them closer to coda and out of the core pipelines, to allow better interaction during the coding process

Reply:
It can be, but where would you like it to go (for now)?

Reply:
For now you can just put it as a private method in the same place. It'll help me in defining the API for the logical imputation

Reply:
Moved to cls._impute_location_codes